### ISSUE TITLE
[Welcoming] Review & troubleshoot a newcomer's pull request

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#3458]
(Work in progress! Please help!)

As part of our ongoing work to welcome newcomers, and to produce more welcoming [first timers only issues](https://code.publiclab.org), we need to help them make well-formatted pull requests, and troubleshoot the issues they encounter.

This will help newcomers get started in a codebase they may not be familiar with! 🔍 🕵️ 

### Find open pull requests

https://github.com/publiclab/plots2/pulls

### See if they're a newcomer

![screen shot 2018-09-25 at 10 39 00 am](https://user-images.githubusercontent.com/24359/46022080-ec3c8880-c0af-11e8-9a1c-7fddb6704111.png)

### Read their Travis errors
![screen shot 2018-09-25 at 10 38 55 am](https://user-images.githubusercontent.com/24359/46021816-720c0400-c0af-11e8-8fb3-cec456e0570c.png)

### Leave inline comments

![screen shot 2018-09-25 at 10 43 40 am](https://user-images.githubusercontent.com/24359/46022136-04140c80-c0b0-11e8-83d8-453099e6bcbd.png)

**Comment by SidharthBansal** [comment]
This task is published on GCI dashboard. Thanks all.

**Comment by JonathanXu1** [comment]
Hello,
I've requested a review and made inline comments for this pull request: https://github.com/publiclab/plots2/pull/3557 . I believe the change would be all that would be needed to further debug the issue. I've submitted this task on GCI. Please let me know if there's anything else I should do.

**Comment by jywarren** [comment]
i'll review it now! Thanks!!!

**Comment by sashadev-sky** [comment]
** Uploaded to GCI dashboard**

**Comment by SidharthBansal** [comment]
Please delete it. This task is broken into weeks and repos before uploading
it. Already done. Thanks

**Comment by sashadev-sky** [comment]
>  This task is broken into weeks and repos before uploading it. Already done. 

Sorry not sure I understand. You're saying a) it should be recreated separately for each repository. b) since we already 1 week into GCI, we should no longer upload it all anyway? Isn't this a supportive task for throughout the whole competition though?

Not sure I have the admin rights to delete it - but you'll find it on the dashboard under the same title as this issue @SidharthBansal

**Comment by SidharthBansal** [comment]
This is already broken by Keshav

**Comment by sashadev-sky** [comment]
oh ok! So we need to add them for every week? So far I see week 1 @SidharthBansal

**Comment by SidharthBansal** [comment]
Yes, we need to create for every week. Week 1 is by Keshav. We are doing
things in rotation. You can take week 5 if you want.

**Comment by sashadev-sky** [comment]
@SidharthBansal ok sure. wouldnt it be easier to just upload them all now and then you publish them as the week comes along?

**Comment by SidharthBansal** [comment]
Yes it is. I think @ishaGupta18 @ananyaarun are busy these days.
I asked them to upload for some weeks. You can upload for week 2 if you
want.
K tasks for k repos. Just copy and paste thing. So it will take 10 min.
Thanks

**Comment by priyda** [comment]
How Can I contribute?
